From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Mon, 7 Mar 2016 23:44:27 +0100
Subject: [PATCH] efi: Add dosfstools to mco container for ESP.

Ubuntu image building for arm64 requires EFI system partition to be
included in partitioning scheme, hence the need for mkfs.vfat.
---
 docker/mcollective/setup.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docker/mcollective/setup.sh b/docker/mcollective/setup.sh
index bfe164c..5fd19f7 100644
--- a/docker/mcollective/setup.sh
+++ b/docker/mcollective/setup.sh
@@ -28,7 +28,7 @@ done
 yum clean expire-cache
 yum update -y
 
-packages="sudo mcollective shotgun fuel-agent fuel-provisioning-scripts psmisc daemonize"
+packages="sudo mcollective shotgun fuel-agent fuel-provisioning-scripts psmisc daemonize dosfstools"
 echo $packages | xargs -n1 yum install -y
 
 # /var/lib/fuel/ibp is a mount point for IBP host volume
-- 
1.9.1

